examples/managers/blender/blender.py

```diff
@@ -0,0 +1,132 @@
+import asyncio
+import json
+import logging.config
+from datetime import timedelta
+from pathlib import Path
+from typing import List
+
+from golem.managers import (
+    ActivityPoolManager,
+    AddChosenPaymentPlatform,
+    AutoDemandManager,
+    LinearAverageCostPricing,
+    MapScore,
+    PayAllPaymentManager,
+    QueueWorkManager,
+    ScoredAheadOfTimeAgreementManager,
+    SequentialNegotiationManager,
+    WorkContext,
+    WorkResult,
+    retry,
+)
+from golem.node import GolemNode
+from golem.payload import RepositoryVmPayload
+from golem.utils.logging import DEFAULT_LOGGING
+
+BLENDER_IMAGE_HASH = "9a3b5d67b0b27746283cb5f287c13eab1beaa12d92a9f536b747c7ae"
+FRAME_CONFIG_TEMPLATE = json.loads(Path(__file__).with_name("frame_params.json").read_text())
+FRAMES = list(range(0, 60, 5))
+
+
+async def run_on_golem(
```

Comment:
again, the construction of the chain of managers seems awfully awkard and fragile part of this process... this may be all right if all we do is keep it as an internal interface but imvho unacceptable for use by external users...

Reply:
We(I and @approxit) discussed a concept of manager stacks which would manage it for the user.